diff --git a/config/strings.conf b/config/strings.conf
index c227a41a..3933ecad 100644
--- a/config/strings.conf
+++ b/config/strings.conf
@@ -204,6 +204,8 @@
 !system 1074 Pendulum
 !system 1075 Special Summon
 !system 1076 Link
+!system 1077 Skill
+!system 1078 Action
 !system 1080 (N/A)
 !system 1081 Extra Monster Zone
 #GUI elements
@@ -243,16 +245,17 @@
 !system 1132 Sealed Duel
 !system 1133 Booster Draft Duel
 !system 1134 Destiny Draw
-!system 1135 Speed Duel
-!system 1136 Concentration Duel
-!system 1137 Boss Duel
-!system 1138 Battle City
-!system 1139 Duelist Kingdom
-!system 1140 Dimension Duel
-!system 1141 Turbo Duel
-!system 1142 Double Deck Duel
-!system 1143 Command Duel
-!system 1144 Deck Masters Duel
+!system 1135 Concentration Duel
+!system 1136 Boss Duel
+!system 1137 Battle City
+!system 1138 Duelist Kingdom
+!system 1139 Dimension Duel
+!system 1140 Turbo Duel
+!system 1141 Double Deck Duel
+!system 1142 Command Duel
+!system 1143 Deck Masters Duel
+!system 1144 Action Duel
+!system 1145 Half Deck Duel
 #Actions
 !system 1150 Activate
 !system 1151 Normal Summon
@@ -279,6 +282,7 @@
 !system 1202 Watch Replay
 !system 1203 AI Mode
 !system 1204 Deck Edit
+!system 1209 Commits
 !system 1210 Exit
 !system 1211 OK
 !system 1212 Cancel
@@ -319,6 +323,8 @@
 !system 1253 Current Spectators:
 !system 1254 Select Deck:
 !system 1255 Ready!
+!system 1258 Speed Duel
+!system 1259 Rush Duel
 !system 1260 Master Rule 1
 !system 1261 Master Rule 2
 !system 1262 Master Rule 3
@@ -359,6 +365,7 @@
 !system 1306 Exit
 !system 1307 Shuffle
 !system 1308 Delete
+!system 1309 Reload
 !system 1310 (All)
 !system 1311 Category:
 !system 1312 Monster
@@ -415,11 +422,14 @@
 !system 1364 Failed to rename deck
 !system 1365 Failed to rename! Maybe there is a replay with same name?
 !system 1366 Restart
+!system 1367 Export finished
 !system 1370 Level ↑
 !system 1371 Attack ↑
 !system 1372 Defense ↑
 !system 1373 Name ↓
 !system 1374 Link Marker
+!system 1380 VS
+!system 1381
 !system 1390 Waiting...
 !system 1391 Waiting....
 !system 1392 Waiting.....
@@ -447,14 +457,32 @@
 !system 1420 There are Extra Deck-only cards in your Main Deck.
 !system 1421 This deck contains card types that are not allowed in the Main Deck.
 !system 1422 "%ls" is a prerelease or unofficial card and cannot be used with the current settings.
+!system 1423 Version mismatch({}.{}).
 !system 1430 Couldn't load ocgcore; you are limited to deck editing, replay watching, and online mode.
 !system 1431 Loaded ocgcore from {}
+!system 1432 Previous
+!system 1433 Next
+!system 1434 History
+!system 1435 URL: {}
+!system 1436 Error: {}
+!system 1439 Chat
+!system 1440 Error
+!system 1441 Viewer
+!system 1442 N/A
+!system 1443 Changelog
+!system 1444 {} FPS
+!system 1445 Show FPS
+!system 1446 Unknown error
+!system 1447 Changelog
+!system 1448 Error
+!system 1449 Failed to load repository.
+!system 1450 Details:
 !system 1500 The duel ended.
 !system 1501 Replay ended.
 !system 1502 Connection has been lost.
-!system 1510 Your choice:[%ls]
-!system 1511 Opponent declared:[%ls]
-!system 1512 Your opponent's choice:[%d]
+!system 1510 Your choice: {}
+!system 1511 Opponent declared: {}
+!system 1512 Your opponent's choice: {}
 !system 1600 Activate a card
 !system 1601 Set a card
 !system 1602 Card control is changed
@@ -472,10 +500,10 @@
 !system 1614 Opponent received %d damage
 !system 1615 Increase %d LP
 !system 1616 Opponent increases %d LP
-!system 1617 "%ls" has placed %d "%ls"
-!system 1618 "%ls" been removed %d "%ls"
-!system 1619 "%ls" Attacks "%ls"
-!system 1620 "%ls" Attacks directly
+!system 1617 "{}" has placed {} "{}"
+!system 1618 "{}" been removed {} "{}"
+!system 1619 "{}" attacks "{}"
+!system 1620 "{}" attacks directly
 !system 1621 Attack Negated
 !system 1622 "%ls" missed the timing
 !system 1623 Coin landed on:
@@ -493,21 +521,58 @@
 !system 1635 Separate Pendulum Zones
 !system 1636 Extra Monster Zones
 !system 1637 Fus/Synch/Xyz anywhere
+!system 1638 Free Trap Monster Zone
+!system 1639 Return to Extra Deck Triggers
+!system 1640 Trigger only in Location
+!system 1641 DUEL_SPSUMMON_ONCE_OLD_NEGATE
+!system 1642 DUEL_CANNOT_SUMMON_OATH_OLD
+!system 1643 No Standby Phase
+!system 1644 No Main Phase 2
+!system 1645 3 Columns
+!system 1646 Draw until 5
+!system 1647 No Hand Limit
+!system 1648 No Summon Limit
+!system 1649 Inverted Quick Priority
+!system 1701 You Win!
+!system 1702 You Lose!
+!system 1703 Draw Game
+!system 1704 Draw Phase
+!system 1705 Standby Phase
+!system 1706 Main Phase
+!system 1707 Battle Phase
+!system 1708 Main Phase 2
+!system 1709 End Phase
+!system 1710 Next Player's Turn
+!system 1711 Duel Start
+!system 1712 Duel 1 Start
+!system 1713 Duel 2 Start
+!system 1714 Duel 3 Start
 !system 1900 OCG
 !system 1901 TCG
 !system 1902 OCG/TCG
 !system 1903 Prerelease
 !system 1904 Anything goes
+!system 1910 Speed Duel
+!system 1911 Rush Duel
 !system 1970 About
 !system 1985 Show started rooms
 !system 1986 started
 !system 1987 open
 !system 1988 Status
+!system 1998 Core Log
 !system 1994 Show locked rooms
 !system 1999 Show unofficial cards
-!system 2000 Old Replay Mode
-!system 2008 Select AI deck
+!system 2000 No log
+!system 2001 Log to chat
+!system 2002 Log to file
+!system 2003 Log to both
 !system 2007 AI script
+!system 2008 Select AI deck
+!system 2009 Turns
+!system 2010 Core: {}.{}
+!system 2011 Expected EDOPro Core Version: {}.{}
+!system 2012 Compatible YGOPro Core Version: {}.{}
+!system 2013 Core: {2} {0}.{1}
 !system 2021 Filter:
 !system 2024 Notes:
 !system 2030 Players:
@@ -515,6 +580,8 @@
 !system 2032 No rooms
 !system 2033 There are no rooms currently open on the server. Click on 'Host' to create a new room or try another server.
 !system 2037 Failed to open url
+!system 2038 Enter the password for the room.
+!system 2040 Version
 !system 2041 Server
 !system 2042 Online
 !system 2043 Expand log
@@ -532,6 +599,27 @@
 !system 2055 AI Level: {}
 !system 2056 Anti-meta AI
 !system 2057 Master Rule supported: {}
+!system 2058 Audio Backend
+!system 2059 Settings
+!system 2060 Fullscreen
+!system 2061 Scale background
+!system 2062 Resize background accurately
+!system 2063 Hide passcode
+!system 2064 Current skin
+!system 2065 No skin
+!system 2066 Reload skin
+!system 2066 Current Locale
+!system 2068 Draw field spells
+!system 2069 Filter bot
+!system 2070 DPI
+!system 2071 Restart
+!system 2072 Show Console
+!system 2073 Enable VSync
+!system 2074 Max FPS
+!system 2075 Anti-Alias
+!system 2076 Show Scope
+!system 2077 Save Hand Test Replays
+!system 2078 Discord
 #Victory Reasons
 !victory 0x0 Surrendered
 !victory 0x1 LP reached 0
