diff --git a/config/strings.conf b/config/strings.conf
index c227a41a..5f4d92bc 100644
--- a/config/strings.conf
+++ b/config/strings.conf
@@ -1,5 +1,9 @@
-#The first line is used for comment
-#line doesn't start with '!' is also neglected
+# Standard Project Ignis English locale file, copyright (C) 2020
+#!system for system strings
+#!victory for victory reasons
+#!counter for counter names
+#!setname for archetype names
+any line that doesn't start with the above keywords is ignored
 
 #System strings
 !system 1 Normal Summon
@@ -24,7 +28,7 @@
 !system 29 End of Battle Phase
 !system 30 Replay, do you want to continue the Battle?
 !system 31 Do you want to Attack Directly?
-!system 40 Damage Step
+!system 40 Start of the Damage Step
 !system 41 Before Damage Calculation
 !system 42 During Damage Calculation
 !system 43 After Damage Calculation
@@ -51,40 +55,40 @@
 !system 97 Place "%ls" in the Spell & Trap Zone?
 !system 100 First to go
 !system 101 Second to go
-!system 102 Self
-!system 103 Opponent
-!system 200 Do you wish to use from "%ls" "%ls"'s effect?
+!system 102 Your 
+!system 103 Your opponent's 
+!system 200 Use the effect of "%ls" from [%ls]?
 !system 201 No effect can be activated at this moment
 !system 202 Do you want to check the field?
 !system 203 Chain another card?
 !system 204 Remove %d "%ls"
 !system 205 Please select a sort order [from top to bottom]
 !system 206 Please select the chain sequence
-!system 207 Revealing %d card(s):
-!system 208 Confirm %d card(s):
+!system 207 Revealing %d card(s): 
+!system 208 Confirm %d card(s): 
 !system 209 The card has met the requisites
 !system 210 Continue selecting cards?
-!system 211 Round count:
-!system 212 Declared card:
-!system 213 Declared type:
-!system 214 Declared attribute:
-!system 215 Declared number:
-!system 216 Activated in chain %d
+!system 211 Turns passed: 
+!system 212 Declared card: 
+!system 213 Declared type: 
+!system 214 Declared attribute: 
+!system 215 Declared number: 
+!system 216 Activated in chain link %d
 !system 217 Target of (chain %d) [%ls]
 !system 218 Pay LP by Effect of [%ls], instead?
-!system 219 Detach XYZ material by Effect of [%ls], instead?
-!system 220 Remove Counter(s) by Effect of [%ls], instead?
-!system 221 on [%ls]. Do you want to activate Trigger Effect of [%ls]?
+!system 219 Detach Xyz Material(s) by the effect of [%ls], instead?
+!system 220 Remove Counter(s) by the effect of [%ls], instead?
+!system 221 Do you want to activate the Trigger Effect of "%ls" from [%ls]?
 !system 222 Activate Trigger Effect?
-!system 223   
-!system 224 First Special Summoned by own Condition
+!system 223  
+!system 224 [Special Summoned properly]
 !system 500 Select a card to tribute
 !system 501 Select a card to discard
 !system 502 Select a card to destroy
 !system 503 Select a card to banish
 !system 504 Select a card to send to Graveyard
 !system 505 Select a card to return to hand
-!system 506 Select the card to add to your hand
+!system 506 Select a card to add to your hand
 !system 507 Select the card(s) to return to Deck
 !system 508 Select the card to Summon
 !system 509 Select a Monster to Special Summon
@@ -92,23 +96,23 @@
 !system 511 Select a card to use as Fusion Material
 !system 512 Select a card to use as Synchro Material
 !system 513 Select a card to use as Xyz Material
-!system 514 Select a card on opponent's side
-!system 515 Select a card on your side
-!system 516 Select a monster in attack position
-!system 517 Select a monster in defense position
+!system 514 Select a face-up card
+!system 515 Select a face-down card
+!system 516 Select a monster in Attack Position
+!system 517 Select a monster in Defense Position
 !system 518 Select a card to equip
-!system 519 Select the Xyz material to detach
+!system 519 Select the Xyz Material to detach
 !system 520 Select the monster to change control
 !system 521 Select the card to replace
-!system 522 Select the monsters in attack
-!system 523 Select the monsters in defense
-!system 524 Select attack monster
-!system 525 Select defense monster
+!system 522 Select a face-up Attack Position monster(s)
+!system 523 Select a face-up Defense Position monster(s)
+!system 524 Select a face-down Attack Position monster(s)
+!system 525 Select a face-down Defense Position monster(s)
 !system 526 Reveal a card
 !system 527 Select the card to place on the field
 !system 528 Select a monster to change its battle position
 !system 529 Select your card
-!system 530 Select opponent's card
+!system 530 Select an opponent's card
 !system 531 Select monsters for Tribute Summon
 !system 532 Select a monster to detach Xyz Material
 !system 533 Select a card to use as Link Material
@@ -140,18 +144,18 @@
 !system 1002 Monster Zone
 !system 1003 Spell/Trap Zone
 !system 1004 Graveyard
-!system 1005 Banished Zone
+!system 1005 Banished
 !system 1006 Extra Deck
 !system 1007 Xyz Material
 !system 1008 Field Spell Zone
 !system 1009 Pendulum Zone
-!system 1010 Earth
-!system 1011 Water
-!system 1012 Fire
-!system 1013 Wind
-!system 1014 Light
-!system 1015 Dark
-!system 1016 Divine
+!system 1010 EARTH
+!system 1011 WATER
+!system 1012 FIRE
+!system 1013 WIND
+!system 1014 LIGHT
+!system 1015 DARK
+!system 1016 DIVINE
 !system 1020 Warrior
 !system 1021 Spellcaster
 !system 1022 Fairy
@@ -204,55 +208,58 @@
 !system 1074 Pendulum
 !system 1075 Special Summon
 !system 1076 Link
+!system 1077 Skill
+!system 1078 Action
 !system 1080 (N/A)
 !system 1081 Extra Monster Zone
-#GUI elements
-!system 1100 S/T Destroy
-!system 1101 Destroy Monster
-!system 1102 Banish
-!system 1103 Graveyard
-!system 1104 Back to Hand
-!system 1105 Back to Deck
-!system 1106 Destroy Hand
-!system 1107 Destroy Deck
+#Deck Edit Categories
+!system 1100 Destroy Monster
+!system 1101 Destroy S/T
+!system 1102 Destroy Deck
+!system 1103 Destroy Hand
+!system 1104 Send to GY
+!system 1105 Send to Hand
+!system 1106 Send to Deck
+!system 1107 Banish
 !system 1108 Draw
 !system 1109 Search
-!system 1110 Recovery
-!system 1111 Position
-!system 1112 Control
-!system 1113 Change ATK/DEF
-!system 1114 Piercing
-!system 1115 Repeat Attack
-!system 1116 Limit Attack
-!system 1117 Direct Attack
-!system 1118 Special Summon
-!system 1119 Token
-!system 1120 Type-Related
-!system 1121 Property-Related
-!system 1122 Damage LP
-!system 1123 Recover LP
-!system 1124 Destroy
-!system 1125 Select
-!system 1126 Counter
-!system 1127 Gamble
-!system 1128 Fusion-Related
-!system 1129 Tuner-Related
-!system 1130 Xyz-Related
-!system 1131 Negate Effect
+!system 1110 Change ATK/DEF
+!system 1111 Change Level/Rank
+!system 1112 Position
+!system 1113 Piercing
+!system 1114 Direct Attack
+!system 1115 Multi Attack
+!system 1116 Negate Activation
+!system 1117 Negate Effect
+!system 1118 Damage LP
+!system 1119 Recover LP
+!system 1120 Special Summon
+!system 1121 Non-effect-related
+!system 1122 Token-related
+!system 1123 Fusion-related
+!system 1124 Ritual-related
+!system 1125 Synchro-related
+!system 1126 Xyz-related
+!system 1127 Link-related
+!system 1128 Counter-related
+!system 1129 Gamble
+!system 1130 Control
+!system 1131 Move Zones
 #Extra Rules
 !system 1132 Sealed Duel
 !system 1133 Booster Draft Duel
 !system 1134 Destiny Draw
-!system 1135 Speed Duel
-!system 1136 Concentration Duel
-!system 1137 Boss Duel
-!system 1138 Battle City
-!system 1139 Duelist Kingdom
-!system 1140 Dimension Duel
-!system 1141 Turbo Duel
-!system 1142 Double Deck Duel
-!system 1143 Command Duel
-!system 1144 Deck Masters Duel
+!system 1135 Concentration Duel
+!system 1136 Boss Duel
+!system 1137 Battle City
+!system 1138 Duelist Kingdom
+!system 1139 Dimension Duel
+!system 1140 Turbo Duel
+!system 1141 Double Deck Duel
+!system 1142 Command Duel
+!system 1143 Deck Masters Duel
+!system 1144 Action Duel
+!system 1145 Deck size limit 20-30
 #Actions
 !system 1150 Activate
 !system 1151 Normal Summon
@@ -274,11 +281,13 @@
 !system 1173 Xyz Summon
 !system 1174 Link Summon
 #Menu strings
-!system 1200 LAN Mode
-!system 1201 Puzzle Mode
-!system 1202 Watch Replay
+!system 1200 LAN + AI
+!system 1201 Puzzles
+!system 1202 Replays
+# UNUSED
 !system 1203 AI Mode
-!system 1204 Deck Edit
+!system 1204 Decks
+!system 1209 Update Log
 !system 1210 Exit
 !system 1211 OK
 !system 1212 Cancel
@@ -298,8 +307,8 @@
 !system 1226 Forbidden List:
 !system 1227 Duel Mode:
 !system 1228 Additional Options
-!system 1229 Don't check deck
-!system 1230 Do not shuffle deck
+!system 1229 Don't check Deck
+!system 1230 Don't shuffle Deck
 !system 1231 Starting LP:
 !system 1232 Starting Hand:
 !system 1233 Cards per Draw:
@@ -316,14 +325,18 @@
 !system 1250 Host
 !system 1251 →Duel
 !system 1252 →Spectate
-!system 1253 Current Spectators: 
+!system 1253 Current Spectators:
 !system 1254 Select Deck:
 !system 1255 Ready!
+!system 1256 Nickname empty
+!system 1257 Please enter a nickname
+!system 1258 Speed Duel
+!system 1259 Rush Duel
 !system 1260 Master Rule 1
 !system 1261 Master Rule 2
 !system 1262 Master Rule 3
 !system 1263 Master Rule 4
-!system 1264 Master Rule 5
+!system 1264 Master Rules (2020)
 !system 1265 Anime/Manga
 !system 1266 Illegal
 !system 1267 Video Game
@@ -332,16 +345,16 @@
 !system 1271 Log
 !system 1272 Clear Log
 !system 1273 Settings
-!system 1274 Auto Monster Card placing
-!system 1275 ↑Random card placing
-!system 1276 Auto Chain order
-!system 1277 No delay for Chain
-!system 1278 Auto Spell/Trap Card placing
+!system 1274 Automatic Monster Card placement
+!system 1275 ↑Randomize card placement
+!system 1276 Automatic chain order
+!system 1277 Add a delay even when no response
+!system 1278 Automatic Spell/Trap Card placement
 !system 1279 Chat
 !system 1280 Standard duel
 !system 1281 Custom
 !system 1282 Clear Chat
-!system 1290 Mute opponent
+!system 1290 Mute opponents
 !system 1291 Mute spectators
 !system 1292 Ignore Chain
 !system 1293 Always Chain
@@ -349,16 +362,17 @@
 !system 1295 Cancel
 !system 1296 Finish
 !system 1297 Test Hand
-!system 1299 Quick Animation
+!system 1299 Quick animations
 !system 1300 Ban List:
 !system 1301 Deck:
 !system 1302 Save
 !system 1303 Save as
 !system 1304 Clear
 !system 1305 Sort
-!system 1306 Exit 
+!system 1306 Exit
 !system 1307 Shuffle
 !system 1308 Delete
+!system 1309 Reset
 !system 1310 (All)
 !system 1311 Category:
 !system 1312 Monster
@@ -383,7 +397,7 @@
 !system 1332 Side:
 !system 1333 Results:
 !system 1334 Side decking completed
-!system 1335 Decks exported!
+!system 1335 Deck saved!
 !system 1336 Scale:
 !system 1337 Delete This Deck?
 !system 1338 Deck Deleted
@@ -402,8 +416,8 @@
 !system 1351 Surrender
 !system 1352 Main message:
 !system 1353 Start at turn:
-!system 1354 Hide Set Names
-!system 1355 Hide Chain Buttons
+!system 1354 Hide archetypes in card display
+!system 1355 Hide chain buttons
 !system 1356 Old Replay Mode
 !system 1357 Load Puzzle
 !system 1358 Export Decks
@@ -415,15 +429,17 @@
 !system 1364 Failed to rename deck
 !system 1365 Failed to rename! Maybe there is a replay with same name?
 !system 1366 Restart
+!system 1367 Decks exported!
 !system 1370 Level ↑
 !system 1371 Attack ↑
 !system 1372 Defense ↑
 !system 1373 Name ↓
 !system 1374 Link Marker
+!system 1380 vs.
+!system 1381 Best of
 !system 1390 Waiting...
 !system 1391 Waiting....
 !system 1392 Waiting.....
-!system 1393 Archetype:
 !system 1400 Unable to connect to the host.
 !system 1401 Connection has been lost.
 !system 1402 An error occurred during the transmission.
@@ -445,16 +461,36 @@
 !system 1418 The size of your Extra Deck should be not more than %d. At the moment its size is %d.
 !system 1419 The size of your Side Deck should be not more than %d. At the moment its size is %d.
 !system 1420 There are Extra Deck-only cards in your Main Deck.
-!system 1421 This deck contains card types that are not allowed in the Main Deck.
+!system 1421 This deck contains card types that are not allowed in the Decks.
 !system 1422 "%ls" is a prerelease or unofficial card and cannot be used with the current settings.
+!system 1423 Outdated client! Please upgrade to {0:d}.{1:d} (ocgcore version {2:d}.{3:d})
 !system 1430 Couldn't load ocgcore; you are limited to deck editing, replay watching, and online mode.
 !system 1431 Loaded ocgcore from {}
+!system 1432 <<<
+!system 1433 >>>
+!system 1434 Error!
+!system 1435 The repo {} couldn't be cloned.
+!system 1436 Error: {}
+!system 1437 Connected to Discord
+!system 1438 Disconnected from Discord
+!system 1439 [System]
+!system 1440 [Script Error]
+!system 1441 [---]
+!system 1442 N/A
+!system 1443 Changelog
+!system 1444 FPS: {}
+!system 1445 Show FPS
+!system 1446 No new commits
+!system 1447 Show full history
+!system 1448 Warning!
+!system 1449 Error while updating repository.
+!system 1450 Make sure you have a working internet connection.
 !system 1500 The duel ended.
 !system 1501 Replay ended.
 !system 1502 Connection has been lost.
-!system 1510 Your choice:[%ls]
-!system 1511 Opponent declared:[%ls]
-!system 1512 Your opponent's choice:[%d]
+!system 1510 Your choice: [{}]
+!system 1511 Opponent declared: [{}]
+!system 1512 Your opponent's choice: [{}]
 !system 1600 Activate a card
 !system 1601 Set a card
 !system 1602 Card control is changed
@@ -465,49 +501,90 @@
 !system 1607 "%ls" Flip Summon...
 !system 1608 Flip Summon success
 !system 1609 "%ls" is activated
-!system 1610 "%ls" (%ls,%d) targeted
+!system 1610 "%ls" (%ls, %d) targeted
 !system 1611 You draw %d card(s)
 !system 1612 Opponent draws %d card(s)
 !system 1613 Received %d damage
 !system 1614 Opponent received %d damage
 !system 1615 Increase %d LP
 !system 1616 Opponent increases %d LP
-!system 1617 "%ls" has placed %d "%ls"
-!system 1618 "%ls" been removed %d "%ls"
-!system 1619 "%ls" Attacks "%ls"
-!system 1620 "%ls" Attacks directly
-!system 1621 Attack Negated
+!system 1617 {2:d} "{1}" has been placed on "{0}"
+!system 1618 {2:d} "{1}" has been removed from "{0}"
+!system 1619 "{}" attacks "{}"
+!system 1620 "{}" attacks directly
+!system 1621 Attack negated
 !system 1622 "%ls" missed the timing
 !system 1623 Coin landed on:
 !system 1624 Die landed on:
 !system 1625 Extra Rules
 !system 1626 Custom
-!system 1627 No %s 
+!system 1627 No %s
 !system 1628 ==Card Type filter==
 !system 1629 ==Game rules==
 !system 1630 Custom Rule
-!system 1631 Obsolete Ignition
-!system 1632 1st Turn Draw
+!system 1631 Ignition Priority
+!system 1632 First turn draw
 !system 1633 1 Field Spell
 !system 1634 Pendulum Zones
 !system 1635 Separate Pendulum Zones
 !system 1636 Extra Monster Zones
-!system 1637 Fus/Synch/Xyz anywhere
+!system 1637 F S X anywhere
+!system 1638 Trap Monsters don't occupy Spell/Trap Zone
+!system 1639 Effects activate face-down in the Extra Deck
+!system 1640 Trigger Effects only activate in the correct location
+!system 1641 Negated summons count for OPT restrictions
+!system 1642 Negated summons count for activation restrictions
+!system 1643 No Standby Phase
+!system 1644 No Main Phase 2
+!system 1645 Three-column field
+!system 1646 Normal draw until 5 in hand
+!system 1647 Unlimited hand size
+!system 1648 Unlimited Normal Summons
+!system 1649 Inverted Quick Priority
+!system 1680 "%ls" (%ls, %d) selected
+!system 1701 You Win!
+!system 1702 You Lose!
+!system 1703 Draw Game
+!system 1704 Draw Phase
+!system 1705 Standby Phase
+!system 1706 Main Phase 1
+!system 1707 Battle Phase
+!system 1708 Main Phase 2
+!system 1709 End Phase
+!system 1710 Next Player's Turn
+!system 1711 Duel Start
+!system 1712 Duel1 Start
+!system 1713 Duel2 Start
+!system 1714 Duel3 Start
 !system 1900 OCG
 !system 1901 TCG
 !system 1902 OCG/TCG
 !system 1903 Prerelease
+!system 1910 Speed
+!system 1911 Rush
 !system 1904 Anything goes
 !system 1970 About
 !system 1985 Show started rooms
 !system 1986 started
 !system 1987 open
 !system 1988 Status
+!system 1989 Request a rematch?
 !system 1994 Show locked rooms
-!system 1999 Show unofficial cards
-!system 2000 Old Replay Mode
-!system 2008 Select AI deck
+!system 1998 Script errors
+!system 1999 Alternate formats
+!system 2000 Don't display
+!system 2001 Display in chat
+!system 2002 Log to file
+!system 2003 Both
+# UNUSED
 !system 2007 AI script
+# UNUSED
+!system 2008 Select AI deck
+!system 2009 Turns
+!system 2010 Loaded ocgcore {}.{}
+!system 2011 Expected ocgcore {}.{}
+!system 2012 Compatibility version {:X}.0{:X}.{:X}
+!system 2013 Loaded ocgcore {}.{} from {}
 !system 2021 Filter:
 !system 2024 Notes:
 !system 2030 Players:
@@ -515,11 +592,13 @@
 !system 2032 No rooms
 !system 2033 There are no rooms currently open on the server. Click on 'Host' to create a new room or try another server.
 !system 2037 Failed to open url
+!system 2038 Enter password:
+!system 2040 Version
 !system 2041 Server
-!system 2042 Online
+!system 2042 Servers
 !system 2043 Expand log
 !system 2044 Shrink log
-!system 2045 Repositories Status
+!system 2045 Repositories
 !system 2046 Enable music
 !system 2047 Enable sound effects
 !system 2048 Music
@@ -532,6 +611,60 @@
 !system 2055 AI Level: {}
 !system 2056 Anti-meta AI
 !system 2057 Master Rule supported: {}
+!system 2058 Failed to initialize audio backend
+!system 2059 More settings (CTRL+O)
+!system 2060 Fullscreen (F11)
+!system 2061 Fit background
+!system 2062 Accurate background resize
+!system 2063 Hide passcodes and scopes in card display
+!system 2064 Skin
+!system 2065 none
+!system 2066 Reload skin
+!system 2067 Language
+!system 2068 Draw Field Spell arts
+!system 2069 Filter available WindBots by Master Rule
+!system 2070 Scale
+!system 2071 Apply and reload
+!system 2072 Windows debug console (requires restart)
+!system 2073 VSync (requires reload)
+!system 2074 FPS limit if VSync off
+!system 2075 MSAA (requires reload)
+!system 2076 Show scope labels on card art
+!system 2077 Save a replay when leaving hand test
+!system 2078 Discord integration
+!system 2100 Yami Yugi
+!system 2101 Ishizu
+!system 2102 Pegasus
+!system 2103 Kaiba
+!system 2104 Joey
+!system 2105 Mai
+!system 2106 Bonz
+!system 2107 Mako
+!system 2108 Rex
+!system 2109 Weevil
+!system 2110 Keith
+!system 2111 Christine
+!system 2112 Emma
+!system 2113 Andrew
+!system 2114 David
+#Puzzle Creator strings
+!system 2200 Add a card?
+!system 2201 Place in a Monster Zone
+!system 2202 Place in a Spell&Trap Zone
+!system 2203 Place in a Pendulum Zone
+!system 2204 Place in a Field Spell Zone
+!system 2205 Place in the GY
+!system 2206 Place in the hand
+!system 2207 Place in the Deck
+!system 2208 Place in the Extra Deck
+!system 2209 Place the card banished
+!system 2210 Have possession over the card?
+!system 2211 Place in face-up position
+!system 2212 Place in face-down position
+!system 2213 Equip?
+!system 2213 Gemini Summon?
+!system 2214 Attach an Xyz Material to this monster? (Max. 5)
+!system 2215 Place in the Extra Monster Zone?
 #Victory Reasons
 !victory 0x0 Surrendered
 !victory 0x1 LP reached 0
@@ -555,6 +688,7 @@
 !victory 0x1e Victory by the effect of Flying Elephant
 !victory 0x1f Victory by the effect of Exodia, Master of the Guard
 !victory 0x20 Match Victory by the effect of 「%ls」
+!victory 0x21 Victory by the effect of True Exodia
 !victory 0x30 Victory by the effect of Creator of Miracles
 !victory 0x51 Victory by the effect of Evil 1
 !victory 0x52 Victory by the effect of Number_Ci1000:_Numerronius_Numerronia
@@ -780,7 +914,7 @@
 !setname 0x44 The Agent
 !setname 0x45 Archfiend
 !setname 0x1045 Red Dragon Archfiend
-!setname 0x46 Polymerization/Fusion
+!setname 0x46 Polymerization|Fusion
 !setname 0x1046 Fusion Dragon
 !setname 0x47 Gem-
 !setname 0x1047 Gem-Knight
@@ -796,7 +930,7 @@
 !setname 0x304e Evoltile
 !setname 0x604e Evolsaur
 !setname 0x504e Evolzar
-!setname 0x4f N/A
+!setname 0x4f Dark Lucius
 !setname 0x104f /Assault Mode
 !setname 0x50 Venom
 !setname 0x1050 Starving Venom
@@ -812,9 +946,9 @@
 !setname 0x5a Penguin
 !setname 0x5b Inmato
 !setname 0x5c Sphinx
-!setname 0x5d N/A
-!setname 0x5e N/A
-!setname 0x5f N/A
+!setname 0x5d Ultimate Insect
+!setname 0x5e Dark Mimic
+!setname 0x5f Mystic Swordsman
 !setname 0x60 Bamboo Sword
 !setname 0x61 Ninjitsu Art
 !setname 0x62 Toon
@@ -835,7 +969,7 @@
 !setname 0x6e Prophecy
 !setname 0x106e Spellbook
 !setname 0x6f Heroic
-!setname 0x106f Heroic Challenger 
+!setname 0x106f Heroic Challenger
 !setname 0x206f Heroic Champion
 !setname 0x70 Chronomaly
 !setname 0x71 Madolche
@@ -860,7 +994,7 @@
 !setname 0x7d Hazy
 !setname 0x107d Hazy Flame
 !setname 0x7e ZW
-!setname 0x7f Utopi/c
+!setname 0x7f Utopic
 !setname 0x107f Utopia
 !setname 0x207f Utopic Future
 !setname 0x80 Duston
@@ -946,7 +1080,7 @@
 !setname 0xbd Gaia The Fierce Knight
 !setname 0xbe Monarch
 !setname 0xbf Charmer
-!setname 0xc0 Familiar-Possessed 
+!setname 0xc0 Familiar-Possessed
 !setname 0xc1 PSY-Frame
 !setname 0x10c1 PSY-Framegear
 !setname 0xc2 Power Tool
@@ -1020,7 +1154,7 @@
 !setname 0xf8 Supreme King
 !setname 0x10f8 Supreme King Gate
 !setname 0x20f8 Supreme King Dragon
-!setname 0xf9 True King
+!setname 0xf9 True Draco|True King
 !setname 0xfa Phantasm Spiral
 !setname 0xfc Gouki
 !setname 0xfb Trickstar
@@ -1063,7 +1197,7 @@
 !setname 0x120 Prank-Kids
 !setname 0x121 Mayakashi
 !setname 0x122 Valkyrie
-!setname 0x123 Rose Dragon
+!setname 0x123 Rose
 !setname 0x1123 Rose Dragon
 !setname 0x124 Machine Angel
 !setname 0x125 Smile
@@ -1085,10 +1219,25 @@
 !setname 0x134 Generaider
 !setname 0x135 @Ignister
 !setname 0x136 A.I.
-!setname 0x137 Senka
+!setname 0x137 Ancient Warriors
 !setname 0x138 Megalith
-!setname 0x139 Onomato
-!setname 0x239 Palladium Oracle
+!setname 0x139 Onomat
+!setname 0x13a Palladium
+!setname 0x13b Rebellion
+!setname 0x13c Codebreaker
+!setname 0x13d Nemesys
+!setname 0x13e Barbaros
+!setname 0x13f Plunder Patroll
+!setname 0x140 Adamancipator
+!setname 0x141 Rikka
+!setname 0x142 Eldlich
+!setname 0x143 Eldlixir
+!setname 0x144 Golden Land
+!setname 0x145 Phantasm
+!setname 0x146 Dragma
+!setname 0x147 Märfae
+!setname 0x148 Potan
+!setname 0x149 Roland
 #Anime Archetypes
 !setname 0x2033 Blackwing Tamer
 !setname 0x5034 Advanced Crystal Beast
@@ -1187,7 +1336,6 @@
 !setname 0x1541 Quiz Panel
 !setname 0x542 Raccoon
 !setname 0x543 Red
-!setname 0x544 Rose
 !setname 0x545 Seal
 !setname 0x546 Shaman
 !setname 0x547 Shark
@@ -1231,7 +1379,7 @@
 !setname 0x561 Yubel
 !setname 0x562 ∞
 !setname 0x563 Yomi
-!setname 0x564 Shogi/ Line Monster
+!setname 0x564 Shogi|Line Monster
 !setname 0x565 Hunder
 !setname 0x566 Heraldic
 !setname 0x567 Hell
@@ -1257,9 +1405,7 @@
 !setname 0x57b Topologina
 !setname 0x580 Stormrider
 !setname 0x581 Drone
-!setname 0x582 @Ignister
-!setname 0x583 Codebreaker
-!setname 0x584 Ai
+!setname 0x583 Mask
 !setname 0x600 Dark Tuner
 !setname 0x601 Dark Synchro
 !setname 0x316 Darkness
