From 89d8ae288405f85803e335415cd709afb5086048 Mon Sep 17 00:00:00 2001
From: LordYuuma <lordyuuma@gmail.com>
Date: Sat, 26 Dec 2020 01:23:34 +0100
Subject: [PATCH] CGUICustomTable: Fix SEGFAULT when lockTexture is missing.

---
 gframe/CGUICustomTable/CGUICustomTable.cpp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/gframe/CGUICustomTable/CGUICustomTable.cpp b/gframe/CGUICustomTable/CGUICustomTable.cpp
index ff00898..597cefb 100644
--- a/gframe/CGUICustomTable/CGUICustomTable.cpp
+++ b/gframe/CGUICustomTable/CGUICustomTable.cpp
@@ -923,13 +923,13 @@ void CGUICustomTable::draw() {
 				driver->draw2DRectangle(skin->getColor(EGDC_HIGH_LIGHT), rowRect, &clientClip);
 
 			const auto lockTexture = driver->getTexture("textures/roombrowser/lock.png");
-			const auto text_size = irr::core::recti(0, 0, lockTexture->getOriginalSize().Width, lockTexture->getOriginalSize().Height);
-			const irr::core::rectf bg_sizef(0, 0, (irr::f32)text_size.getWidth(), (irr::f32)text_size.getHeight());
 			for(u32 j = 0; j < Columns.size(); ++j) {
 				textRect.UpperLeftCorner.X = pos + CellWidthPadding;
 				textRect.LowerRightCorner.X = pos + Columns[j].Width - CellWidthPadding;
 				if((uintptr_t)Rows[i].Items[j].Data == 1) {
 					if(lockTexture) {
+						const auto text_size = irr::core::recti(0, 0, lockTexture->getOriginalSize().Width, lockTexture->getOriginalSize().Height);
+						const irr::core::rectf bg_sizef(0, 0, (irr::f32)text_size.getWidth(), (irr::f32)text_size.getHeight());
 						auto temp = textRect;
 						temp.UpperLeftCorner += irr::core::vector2di(2, 2);
 						temp.LowerRightCorner -= irr::core::vector2di(2, 2);
@@ -1286,4 +1286,3 @@ void CGUICustomTable::deserializeAttributes(io::IAttributes* in, io::SAttributeR
 } // end namespace irr
 
 #endif
-
--
libgit2 1.1.0

