From 3404586bf6795b075bccb4ec106544d29a08ca35 Mon Sep 17 00:00:00 2001
From: edo9300 <edoardo762@gmail.com>
Date: Sat, 2 May 2020 13:49:05 +0200
Subject: [PATCH] Put back wrongly removed parameters

---
 gframe/game.cpp | 14 +++++++-------
 ocgcore         |  2 +-
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/gframe/game.cpp b/gframe/game.cpp
index 6d4fa45..97f9f3e 100644
--- a/gframe/game.cpp
+++ b/gframe/game.cpp
@@ -1680,15 +1680,15 @@ bool Game::MainLoop() {
 					discord.UpdatePresence(DiscordWrapper::DUEL);
 			}
 			if (showcardcode == 1 || showcardcode == 3)
-				gSoundManager->PlayBGM(SoundManager::BGM::WIN);
+				gSoundManager->PlayBGM(SoundManager::BGM::WIN, gGameConfig->loopMusic);
 			else if (showcardcode == 2)
-				gSoundManager->PlayBGM(SoundManager::BGM::LOSE);
+				gSoundManager->PlayBGM(SoundManager::BGM::LOSE, gGameConfig->loopMusic);
 			else if (dInfo.lp[0] > 0 && dInfo.lp[0] <= dInfo.lp[1] / 2)
-				gSoundManager->PlayBGM(SoundManager::BGM::DISADVANTAGE);
+				gSoundManager->PlayBGM(SoundManager::BGM::DISADVANTAGE, gGameConfig->loopMusic);
 			else if (dInfo.lp[0] > 0 && dInfo.lp[0] >= dInfo.lp[1] * 2)
-				gSoundManager->PlayBGM(SoundManager::BGM::ADVANTAGE);
+				gSoundManager->PlayBGM(SoundManager::BGM::ADVANTAGE, gGameConfig->loopMusic);
 			else
-				gSoundManager->PlayBGM(SoundManager::BGM::DUEL);
+				gSoundManager->PlayBGM(SoundManager::BGM::DUEL, gGameConfig->loopMusic);
 			MATERIAL_GUARD(
 			DrawBackImage(imageManager.tBackGround, resized);
 			DrawBackGround();
@@ -1703,15 +1703,15 @@ bool Game::MainLoop() {
 				discord.UpdatePresence(DiscordWrapper::DECK_SIDING);
 			else
 				discord.UpdatePresence(DiscordWrapper::DECK);
-			gSoundManager->PlayBGM(SoundManager::BGM::DECK);
+			gSoundManager->PlayBGM(SoundManager::BGM::DECK, gGameConfig->loopMusic);
 			DrawBackImage(imageManager.tBackGround_deck, resized);
 			MATERIAL_GUARD(DrawDeckBd());
 		} else {
 			if(dInfo.isInLobby)
 				discord.UpdatePresence(DiscordWrapper::IN_LOBBY);
 			else
 				discord.UpdatePresence(DiscordWrapper::MENU);
-			gSoundManager->PlayBGM(SoundManager::BGM::MENU);
+			gSoundManager->PlayBGM(SoundManager::BGM::MENU, gGameConfig->loopMusic);
 			DrawBackImage(imageManager.tBackGround_menu, resized);
 		}
 #ifndef __ANDROID__
--
libgit2 1.0.0

